src/main/scala/com/kotori316/fluidtank/Config.scala

```diff
@@ -42,7 +42,7 @@ object Config {
     val enableFluidSupplier = builder.comment("True to allow fluid supplier to work.").define("enableFluidSupplier", true)
 
     val enablePipeRainbowRenderer = builder.worldRestart().comment("False to disable rainbow renderer for pipe.").define("enablePipeRainbowRenderer", true)
```

Comment:
```suggestion
    val enablePipeRainbowRenderer = builder.worldRestart().comment("False to disable rainbow renderer for pipe.").define("enablePipeRainbowRenderer", false)
```
Pipe rainbow should be disabled by default.